app/controllers/application_controller.rb

```diff
@@ -41,6 +43,9 @@ class ApplicationController < ActionController::API
   def with_current_request
     Insights::API::Common::Request.with_request(request) do |current|
       begin
+        if Tenant.tenancy_enabled? && Insights::API::Common::RBAC::Access.enabled?
+          raise RbacError unless current.user.org_admin?
+        end
         if Tenant.tenancy_enabled? && current.required_auth?
```

Comment:
Can we just collapse this check for `Tenant.tenancy_enabled?` here and above into a single check?